subprojects/scala/src/integTest/groovy/org/gradle/scala/compile/CachedScalaCompileIntegrationTest.groovy

```diff
@@ -39,11 +39,11 @@ class CachedScalaCompileIntegrationTest extends AbstractCachedCompileIntegration
             }
 
             dependencies {
-                compile group: 'org.scala-lang', name: 'scala-library', version: '2.11.8'
+                compile group: 'org.scala-lang', name: 'scala-library', version: '2.11.11'
```

Comment:
I wouldn't do that now. What I remember is that we stopped upgrading Groovy versions all over the place, right? Do you think we should want to set one Scala version in a central place and then always use this one?